README.md

```diff
@@ -11,23 +11,23 @@ The [MySQL](https://www.mysql.com/) operator provides an open-source relational
 ## Deployment instructions
 
 
-Create a Juju model for your operators, say "mysql"
+Create a Juju model for your operators, say "mysql-k8s"
 
 ```bash
-$ juju add-model mysql
+$ juju add-model mysql-k8s
 ```
 
 Deploy a single unit of MySQL using its default configuration
 
 ```bash
-$ juju deploy ./mysql.charm --resource mysql-image=ubuntu/mysql:latest
+$ juju deploy ./mysql-k8s.charm --resource mysql-image=ubuntu/mysql:latest
```

Comment:
Shouldn't this be written as if deploying from charmhub?

Reply:
I was thinking in changing this once the charm is uploaded to charmhub